docs/migration_guide.rst

```diff
@@ -3,8 +3,8 @@ X.X.X migration guide
 
 TODO(Update this with correct version or move to separate file)
```

Comment:
We can't forget about those todos. I've noticed that the old comment is resolved. Just reminding 😅